drivers/display/display_st7735r.c

```diff
@@ -0,0 +1,461 @@
+/*
+ * Copyright (c) 2020 Kim Bøndergaard <kim@fam-boendergaard.dk>
```

Comment:
It seems to be based on display_st7789v.c, so please respect the copyrights of the authors.

Reply:
I certainly do not want to be unrespectful but how is this usually done?
display_st789v.c contains 5 different copyright lines. Am I supposed to keep those 5 and add yet a line? Or am I supposed to not even list myself?
What's the rules for 'copyrights'. I mean even the Kconfigs has copyright notices, even though you can claim they are all copies of each other (with minor changes)

Reply:
@jfischer-no Is a line `Based on display_st7789v.c` OK for you or do you would like to see all copyrights copied?

Reply:
Well the author started with the copy of `drivers/display/display_st7789v.c`, right? IMHO, it would be more correct to list them all.